lib/node_modules/@stdlib/number/float32/base/normalize/README.md

```diff
@@ -145,6 +145,95 @@ for ( i = 0; i < 100; i++ ) {
 
 <!-- /.examples -->
 
+<!-- C interface documentation. -->
+
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/number/float32/base/normalize.h"
+```
+
+#### stdlib_base_float32_normalize( x, \*y, \*exp )
+
+Returns a normal number `y` and exponent `exp` satisfying `x = y * 2^exp`.
+
+```c
+#include <stdint.h>
+float y;
+int32_t exp;
+stdlib_base_float32_normalize( 3.14, &y, &exp );
+```
+
+The function accepts the following arguments:
+
+-   **x**: `[in] float` input value.
+-   **y**: `[out] float*` destination for output value.
+-   **exp**: `[out] int32_t*` destination for exponent.
+
+```c
+void stdlib_base_float32_normalize( const float x, float *y, int32_t *exp );
+```
+
+</section>
+
+<!-- /.usage -->
+
+<!-- C API usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="notes">
+
+</section>
+
+<!-- /.notes -->
+
+<!-- C API usage examples. -->
+
+<section class="examples">
+
+### Examples
+
+```c
+#include "stdlib/number/float32/base/normalize.h"
```

Comment:
After looking at the `example.c` file, that example is correct. Its contents should be copied over to here.